cmd/oci-create-runtime-bundle/main.go

```diff
@@ -130,6 +130,10 @@ func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
 
 	case image.TypeImage:
 		err = image.CreateRuntimeBundle(args[0], args[1], v.ref, v.root)
+
+	default:
+		err = fmt.Errorf("cannot create %q", v.typ)
```

Comment:
For the two I feel are almost, but according to the above test and [other uses](https://github.com/opencontainers/image-tools/blame/master/cmd/oci-image-validate/main.go#L176) ， %q is more appropriate point